packages/relay-signer/src/web3/sender.ts

```diff
@@ -30,17 +35,18 @@ export type DefenderRelaySenderOptions = Partial<{
   maxPriorityFeePerGas: BigUInt;
   speed: Speed;
   validForSeconds: number;
+  authConfig: AuthConfig;
```

Comment:
do we need authConfig here? used for logic from web3 module?

Reply:
yup, it's a leftover